pkg/operatorsource/configuring_test.go

```diff
@@ -9,12 +9,108 @@ import (
 
 	gomock "github.com/golang/mock/gomock"
 	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	"github.com/operator-framework/operator-marketplace/pkg/appregistry"
+	"github.com/operator-framework/operator-marketplace/pkg/datastore"
 	mocks "github.com/operator-framework/operator-marketplace/pkg/mocks/operatorsource_mocks"
 	"github.com/operator-framework/operator-marketplace/pkg/operatorsource"
 	"github.com/operator-framework/operator-marketplace/pkg/phase"
 	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
 )
 
+func TestReconcile_ScheduledForDownload_Success(t *testing.T) {
```

Comment:
I realize the original did not have a doc. But it would help if a doc was added here describing the test case. I also think the name should be changed.